apps/gallery/js/thumbnails.js

```diff
@@ -21,7 +21,9 @@
   // Bug 1135256: Having a listener in ThumbnailList prevents them from
   // being garbage collected since ready() holds a strong reference, so
   // we have a singleton listener here.
-  navigator.mozL10n.ready(thumbnails.list.localize.bind(thumbnails.list));
+  thumbnails.list.localize();
```

Comment:
Actually, I don't think this call to localize() is required at all. The individual thumbnails and the group headers call their own localize() method from their constructors. I don't know when that change happened, but it means that we only need to call this top-level localize() function no DOMRetranslated, I think.  So I think you can just remove this line.